maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/mapper/FinancialAssessmentMapper.java

```diff
@@ -46,4 +57,12 @@ public interface FinancialAssessmentMapper {
 
     FinAssIncomeEvidenceDTO finAssIncomeEvidenceEntityToFinAssIncomeEvidenceDTO(final FinAssIncomeEvidenceEntity finAssIncomeEvidenceEntity);
 
-   }
\ No newline at end of file
+    default IOJAssessorDetails createIOJAssessorDetails(FinancialAssessmentEntity financialAssessment) {
```

Comment:
Please change createIOJAssessorDetails to createMeansAssessorDetails

Reply:
done :)

```
    default AssessorDetails createMeansAssessorDetails(FinancialAssessmentEntity financialAssessment) {
        String fullName = UserEntityUtils.extractFullName(financialAssessment.getUserCreatedEntity());

        return AssessorDetails.builder()
                .fullName(fullName)
                .userName(financialAssessment.getUserCreated())
                .build();
    }
```